src/lib.rs

```diff
@@ -91,10 +91,14 @@ pub struct Assert {
     cmd: Vec<String>,
     expect_success: bool,
     expect_exit_code: Option<i32>,
-    expect_output: Option<String>,
-    fuzzy_output: bool,
-    expect_error_output: Option<String>,
-    fuzzy_error_output: bool,
+    expect_stdout: Option<OutputAssertion>,
+    expect_stderr: Option<OutputAssertion>,
+}
+
+#[derive(Debug)]
+struct OutputAssertion {
+    expect: String,
+    fuzzy: bool,
```

Comment:
Hm, I think I'll just open a new PR where I try that design after this one gets merged. Issues that are not about bugs or features but contain a ton of quoted code are kinda hard to keep track of :)